commitizen/commands/bump.py

```diff
@@ -170,7 +170,10 @@ def __call__(self):  # noqa: C901
             out.write(information)
 
         if increment is None and new_tag_version == current_tag_version:
-            raise NoneIncrementExit()
+            raise NoneIncrementExit(
+                "[NO_COMMITS_TO_BUMP]\n"
```

Comment:
nitpick: do we need the header here?

Reply:
Because it's raising the same error code as no commits found, I thought about outputing similar error with information

Reply:
got it. Thanks! Then, I think we're ready to merge it 🎉